TestFoundation/TestNSString.swift

```diff
@@ -753,50 +754,6 @@ class TestNSString : XCTestCase {
         }
     }
     
-    func test_deletingLastPathComponent() {
```

Comment:
As above, don't delete this method and move it elsewhere in the file. If you want to keep the new test with this one, insert it above here.